src/ui/Languages/es-AR.xml

```diff
@@ -347,7 +362,100 @@ Nota: comprobar el espacio libre en disco.</WaveFileMalformed>
+
+¿Deseas continuar?</ResetWarning>
+    <CreateSimpleTitle>Crear simple</CreateSimpleTitle>
+    <CreateSimpleInstruction>Ingresa estas reglas básicas y el perfil actual se actualizará en consecuencia.</CreateSimpleInstruction>
+    <CreateSimpleGapInstruction>La cantidad mínima de espacio entre subtítulos.</CreateSimpleGapInstruction>
+    <CreateSimpleInCues>Las señales de entrada deben ser:</CreateSimpleInCues>
+    <CreateSimpleInCues0Frames>En el cambio de toma</CreateSimpleInCues0Frames>
+    <CreateSimpleInCues1Frames>1 cuadro después del cambio de toma</CreateSimpleInCues1Frames>
+    <CreateSimpleInCues2Frames>2 cuadros después del cambio de toma</CreateSimpleInCues2Frames>
+    <CreateSimpleInCues3Frames>3 cuadros después del cambio de toma</CreateSimpleInCues3Frames>
+    <CreateSimpleOutCues>Las señales de salida deben ser:</CreateSimpleOutCues>
+    <CreateSimpleOutCues0Frames>En el cambio de toma</CreateSimpleOutCues0Frames>
+    <CreateSimpleOutCues1Frames>1 cuadro antes del cambio de toma</CreateSimpleOutCues1Frames>
+    <CreateSimpleOutCues2Frames>2 cuadros antes del cambio de toma</CreateSimpleOutCues2Frames>
+    <CreateSimpleOutCues3Frames>3 cuadros antes del cambio de toma</CreateSimpleOutCues3Frames>
+    <CreateSimpleOutCuesGap>Espacio mínimo antes del cambio de toma</CreateSimpleOutCuesGap>
+    <CreateSimpleSnapClosestCue>Para subtítulos conectados, ajustar la señal de entrada o salida a un cambio de toma según cuál sea la más cercana</CreateSimpleSnapClosestCue>
+    <CreateSimpleMaxOffset>Desplazamiento máximo:</CreateSimpleMaxOffset>
+    <CreateSimpleMaxOffsetInstruction>Las señales dentro de esta distancia del cambio de toma se ajustarán al cambio de toma.</CreateSimpleMaxOffsetInstruction>
+    <CreateSimpleSafeZone>Zona segura:</CreateSimpleSafeZone>
+    <CreateSimpleSafeZoneInstruction>Las señales dentro de esta distancia del cambio de toma se alejarán del cambio de toma.</CreateSimpleSafeZoneInstruction>
+    <CreateSimpleChainingGap>Intervalo máximo de encadenamiento:</CreateSimpleChainingGap>
+    <CreateSimpleChainingGapInstruction>Si el espacio entre dos subtítulos es menor que esta cantidad, los subtítulos estarán encadenados.</CreateSimpleChainingGapInstruction>
+    <CreateSimpleChainingGapAfterShotChanges>Después de una señal de salida en un cambio de toma, el espacio puede ser menor</CreateSimpleChainingGapAfterShotChanges>
+    <CreateSimpleChainingToolTip>Se recomienda encadenar subtítulos para garantizar un "ritmo" constante en el "parpadeo" de los subtítulos.
+Esto ofrece una experiencia de visualización más relajada.
+
+Después del encadenamiento, los subtítulos estarán conectados (es decir, un subtítulo desaparece y un nuevo subtítulo aparece inmediatamente después de una pequeña pausa) o no.
+Esto le da al espectador una idea de cuándo puede volver a centrar su atención en la pantalla.
+
```

Comment:
Is the line break okay here?

Reply:
> I think so. Is it not? 🤔 This was the original text:
> 
> ```
> <CreateSimpleChainingToolTip>Chaining subtitles is recommended to ensure a consistent "rhythm" in the "flashing" of the subtitles.
> This offers a more relaxed viewing experience.
> 
> After chaining, subtitles are either connected (i.e. a subtitle disappears and a new subtitle appears immediately after a slight pause) or not.
> This gives the viewer some sense on when they can shift their focus back to the screen.
> 
> The length of the chaining gap can be a bit smaller right after a subtitle disappears on a shot change, because the changing shot "resets" the image in a way.
> We leverage the intrinsic rhythm of the image.</CreateSimpleChainingToolTip>
> ```

It's then, just wanted to make sure :)